### ISSUE TITLE
Add params_to_tune for pf NNs (`DeepARModel`, `TFTModel`)

### ISSUE THREAD

**Issue by Mr-Geekman** [tinkoff-ai/etna#1197]
### 🚀 Feature Request

Add params_to_tune for pf NNs (`DeepARModel`, `TFTModel`)

### Proposal

Suggested grid for `DeepARModel`:
```
{
    "hidden_size": IntDistribution(4, 64, step=4),
    "rnn_layers": IntDistribution(4, 3, step=1),
    "dropout": Uniform(0, 0.5),
    "lr": LogUniformDistribution(1e-5, 1e-2),
}
```

Suggested grid for `TFTModel`:
```
{
    "hidden_size": IntDistribution(4, 64, step=4),
    "lstm_layers": IntDistribution(4, 3, step=1),
    "dropout": Uniform(0, 0.5),
    "attention_head_size": IntDistribution(2, 8, step=2),
    "lr": LogUniformDistribution(1e-5, 1e-2),
}
```

To discuss:
- Is tuning `lr` necessary?

### Test cases

Look at https://github.com/tinkoff-ai/etna/issues/1184.

**Comment by martins0n** [comment]
The same comments for lr as for mlp rnn

**Comment by Mr-Geekman** [comment]
Closed by #1210.
